website/docs/reference/global-configs/logs.md

```diff
@@ -131,7 +131,7 @@ Just like other global configs, it is possible to override these values for your
 
 ### Suppress non-error logs in output
 
-By default, dbt shows all logs in standard out (stdout). You can use the `QUIET` config to show only error logs in stdout. Logs will still include the output of anything passed to the `print()` macro.  For example, you might suppress all but error logs to more easily find and debug a jinja error.
+By default, dbt shows all logs in standard out (stdout). You can use the `QUIET` config to show only error logs in stdout. Logs will still include the output of anything passed to the [`print()`](/reference/dbt-jinja-functions/print) macro.  For example, you might suppress all but error logs to more easily find and debug a jinja error.
```

Comment:
This text will not be part of the suggestion. It's where you can elaborate on your suggestion or ask clarifying questions

```suggestion
This is what a suggestion looks like when you add the comment
```